Windows File Path Generated in macOS

**Describe the bug**
There seems to be a Windows-style file path generated by the project when running on OSX: `./src/Foundation/App_Data/Foundation\SearchIndex` 

The error generated when trying to run with this project structure is the following:
`/Applications/Visual Studio.app/Contents/Resources/lib/monodevelop/bin/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(3177,5): error MSB3552: Resource file "**/*.resx" cannot be found.`

**To Reproduce**
Steps to reproduce the behavior:
1. Clone and install dependencies
2. Run `sql_server_optimizely` docker container
3. Run VS Solution
4. See error

**Expected behavior**
The file path generated should be OS-agnostic. If not, documentation on the workaround should be provided.

**Desktop (please complete the following information):**
 - OS: 12.1 macOS
 - Browser: Chrome
 - Version: `main` branch of Foundation. Seems to be `v2021.10.1` based on tags.

--- a/src/Foundation/appsettings.json
+++ b/src/Foundation/appsettings.json
@@ -20,7 +20,7 @@
       "SearchOptions": {
         "DefaultSearchProvider": "LuceneSearchProvider",
         "MaxHitsForSearchResults": 1000,
-        "IndexerBasePath": "[appDataPath]\\Foundation\\SearchIndex",
+        "IndexerBasePath": "[appDataPath]/Foundation/SearchIndex",
         "IndexerConnectionString": "",
         "SearchProviders": [
           {
@@ -28,7 +28,7 @@
             "Type": "Mediachase.Search.Providers.Lucene.LuceneSearchProvider, Mediachase.Search.LuceneSearchProvider",
             "Parameters": {
               "queryBuilderType": "Mediachase.Search.Providers.Lucene.LuceneSearchQueryBuilder, Mediachase.Search.LuceneSearchProvider",
-              "storage": "[appDataPath]\\SearchIndex",
+              "storage": "[appDataPath]/SearchIndex",
               "simulateFaceting": "true"
             }
           }
@@ -45,7 +45,7 @@
       }
     },
     "FindCommerce": {
-      "IgnoreWebExceptionOnInitialization" :  true
+      "IgnoreWebExceptionOnInitialization": true
     }
   }
 }

